websites/visualization/src/app/react/experimental/await/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { ErrorBoundary, Suspense } from '@suspensive/react'
+import { Await } from '@suspensive/react/experimental'
+
+const delay = (ms: number) =>
+  new Promise((resolve) =>
+    setTimeout(() => {
+      resolve('resolved')
+    }, ms)
+  )
+
+const asyncFn = () => delay(4000 * Math.random()).then(() => 'success' as const)
+
+export default function Page() {
+  return (
+    <ErrorBoundary fallback={() => <>error</>}>
+      <Suspense fallback={<>loading...</>}>
+        <Await options={{ key: [2000] as const, fn: asyncFn }}>
+          {(awaited) => (
+            <div>
+              <div>{awaited.data}</div>
+              <div>{awaited.data}</div>
+            </div>
+          )}
+        </Await>
```

Comment:
Will children be invisible before `asyncFn` is resolved? 
Does the external `Suspense` handle the loading(pending) process?

Reply:
Yeah, That's right! and this will be cached in cachekey. we can reuse